src/main/kotlin/se/uulm/snowballr/backend/repository/ProjectTableRepo.kt

```diff
@@ -62,6 +74,21 @@ interface IProjectTableRepo {
             ProjectStatus.PROJECT_STATUS_ACTIVE_LOCKED,
         ),
     ): List<Project>
+
+    /**
+     * Updates an existing project in the database with the provided new information.
+     * The following fields can be updated:
+     * - name
+     * - status
+     * - similarity_threshold
+     * - snowballing_type
+     * - review_maybe_allowed
+     *
+     * @param request The update request containing the new project details, such as the new name.
+     * @param isActiveLocked True, if the project is of status [ProjectStatus.PROJECT_STATUS_ACTIVE_LOCKED] (only the name and status can be updated then).
```

Comment:
Exceeded the maximum line length.
```suggestion
     * @param isActiveLocked True, if the project is of status [ProjectStatus.PROJECT_STATUS_ACTIVE_LOCKED] (only the
     * name and status can be updated then).
```